packages/asc-ui/src/utils/grid.ts

```diff
@@ -80,12 +80,17 @@ export const margin = (layoutId: string, withUnit = false) => ({
 /**
  * Get the min-width breakpoint value for a specific layout
  */
-export const min = (layoutId: string, withUnit = false) => ({
+export const min = (
+  layoutId: string,
+  withUnit = false,
+  noConflict = false,
+) => ({
   theme,
 }: {
   theme: Theme.ThemeInterface
 }): string | number | undefined => {
-  const minValue = fromTheme(`layouts.${layoutId}.min`)({ theme })
+  let minValue = fromTheme(`layouts.${layoutId}.min`)({ theme })
+  minValue += noConflict ? 0.02 : 0
```

Comment:
Where is this value of 0.02 based on?

Reply:
0.02 is a standard used value by bootstrap for instance. see also https://stackoverflow.com/questions/51566916/why-does-bootstrap-use-a-0-02px-difference-between-screen-size-thresholds-in-its